rfcs/text/0000-audr-analysis-updates.md

```diff
@@ -0,0 +1,117 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC
+number and this pull request in the format:*
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# AUDR Analysis Updates
+
+## Summary
+Enable analysis to update previous analyses and add new analyses alongside existing analyses.
+
+## Author(s)
+[Justin Clark-Casey](mailto:justincc@ebi.ac.uk)
+
+## Shepherd
+[Parth Shah](mailto:pshah@chanzuckerberg.com)
+
+## Motivation
+This covers [use cases](https://docs.google.com/document/d/1rI8PCASomdAHznyWQceRJGv4-wtg-P5Rm-rI_uJLjtE/edit#):
+
+* **Fix an incorrect metadata field (e.g. cell count)** where the change can trigger re-analysis.
+* **Fix an incorrect ontology term** where the change can trigger re-analysis.
+* Fix incorrect analysis using a corrected pipeline.
+* **Add better analysis using an improved pipeline.**
+
+Wranglers consider the **bolded** use cases in red to be critical for general availability.
```

Comment:
Can we remove the reference to GA? We are already technically in "GA" and this label means different things to different people.